src/components/RangeSlider/components/DualThumb/DualThumb.scss

```
@@ -83,42 +92,56 @@ $range-thumb-shadow-focus: 0 0 0 rem(1px) color('indigo');
 .Thumbs {
   position: absolute;
   z-index: z-index('input', $stacking-order);
+  padding: 0;
   width: $range-thumb-size;
   height: $range-thumb-size;
   border-radius: 50%;
-  border: border-width() solid color('sky', 'lighter');
+  border: border-width() solid
+    var(--p-action-interactive, color('sky', 'lighter'));
```

This has recently changed to --p-interactive
```suggestion
    var(--p-interactive, color('sky', 'lighter'));
```